docs/compute-element/hosted-ce.md

```diff
@@ -0,0 +1,135 @@
+    so it must also have permissions to submit jobs to the batch system
+
+-   If your batch system is not HTCondor,
+    there must be a shared file system between the submit server and the compute nodes
+
+
+Which Science Communities and Institutions am I Supporting?
+-----------------------------------------------------------
+
+The OSG provides monitoring to view which communities are accessing your site, their fields of science, and home institution.
+Below is an example of the monitoring views that will be available for your cluster.
+
+![monitoring graphs](/compute-element/img/monitoring_graphs.png)
+
+
+Security
+--------
+
+OSG takes multiple precautions to maintain security and prevent unauthorized
+usage of resources:
+
+-   Access to the OSG system with SSH keys are restricted to the OSG staff maintaining them
+-   Users are carefully vetted before they are allowed to submit jobs to OSG
+-   Jobs running through OSG can be traced back to the user that submitted them
+-   Job submission can quickly be disabled if needed
+-   OSG staff are readily contactable in case of an emergency,
+    through email at <mailto:help@opensciencegrid.org>
+
+
+Applying for an OSG Hosted CE
+-----------------------------
+
+Before making any system changes, you should do the following steps:
+
+-   Fill out the [cluster integration questionnaire](http://goo.gl/forms/8OukxsyG6KBSGHuR2)
+    so that the OSG team has basic information about your cluster
+
+-   Email [help@opensciencegrid.org](mailto:help@opensciencegrid.org)
+    to set up a consultation call with the OSG team,
+    in order to discuss how you would like to contribute to the OSG;
+    for example, the number of OSG jobs that should run, what resource limits you have,
+    or which science communities you support
+
+After the consultation, do the following:
+
+-   Create a Unix account on the submit server for use by the OSG CE
+
+-   Install public SSH keys for the account
+
+Once this is done, the OSG team will:
+
+-   Configure the OSG CE with your system details
+
+-   Validate operation with a set of test jobs
+
+-   Configure central OSG services to schedule jobs
+
+Optionally, we can assist you in installing and setting up the Squid and OASIS software on your cluster
+in order to support application software repositories.
+This will allow a broader set of jobs to run on your cluster.
+
+
+Technical Setup
+---------------
+
+An OSG Hosted CE site needs to have access to the worker node client.
+On a Hosted CE with a non-HTCondor batch system, this is often done by unpacking the worker node client tarball into the
+users home directories, if this is on a shared filesystem.
+But for sites with an HTCondor batch system, often there is no shared filesystem set up.
+In that case, the site can either set up a shared filesystem, or install the OSG worker node client.
+This can be obtained from the osg-wn-client rpm, the osg-wn tarball, or used directly from OASIS.
```

Comment:
Even though it's the next section, turn OASIS into a link to it

Reply:
We should also say something along the lines of: if it is easier for your site to install and maintain the WN client than to set up a shared file system, you have the following options: